schedule/yast/sle/guided_xfs/guided_xfs_x86_64.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name:           guided_xfs
+description:    >
+  Guided Partitioning installation with xfs filesystem.
+vars:
+  YUI_REST_API: 1
+schedule:
+  extension_module_selection:
+    - installation/module_registration/register_module_desktop
+  system_role:
+    - installation/system_role/accept_selected_role_SLES_with_GNOME
+  guided_partitioning:
+    - installation/partitioning/select_guided_setup
+    - installation/partitioning/guided_setup/accept_default_part_scheme
+    - installation/partitioning/guided_setup/select_filesystem_option_xfs
+    - installation/partitioning/accept_proposed_layout
```

Comment:
@jknphy Is [this](https://openqa.suse.de/tests/10458327#step/accept_proposed_layout#1/2) expected? I thought a section in schedule file was overriding the default, but here it seems like what is in default is appended at the end.